pom.xml

```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
-            <version>4.10</version>
+            <version>4.11</version>
```

Comment:
Do we want to update the JUnit version? Is this intentional? I normally like to avoid package changes if possible, but if we need to bump it that's fine